### ISSUE TITLE
cURL errors are not caught and thrown as an user exception

### ISSUE THREAD

**Issue by kachnitel** [keboola/generic-extractor#11]
```
(2) [2016-02-25 10:22:07] ex-generic-v2.ERROR: cURL error 6: Could not resolve host: .zendesk.com; Name or service not known {"er ... ass":"Keboola\\GenericExtractor\\Executor","type":"->","args":[]}],"exception":"GuzzleHttp\\Exception\\RequestException"} []
```

**Comment by kachnitel** [comment]
https://papertrailapp.com/groups/23635/events?q=523b5c8295a47f6edf4483dbc6e607bb

**Comment by ErikZigo** [comment]
Fixed in 27fb27684026b3fd0ba70af6dfd955c49fa48086 (0.6.17)
